src/style.css

```diff
@@ -1,3 +1,84 @@
 body {
   margin: 0;
+  font-family: Roboto, Arial, sans-serif;
+  box-sizing: border-box;
+
+  --blue-color: #00acdc;
+}
+
+@font-face {
+  font-family: Roboto;
+  src: url(fonts/Roboto-Medium.ttf);
```

Comment:
The path to the font file seems to be incorrect. The correct path should be `src: url(../fonts/Roboto-Medium.ttf);`.